build/azure-pipelines/linux/product-build-linux.yml

```diff
@@ -189,12 +189,34 @@ steps:
         yarn gulp vscode-linux-$(VSCODE_ARCH)-min-ci
         ARCHIVE_PATH=".build/linux/client/code-${{ parameters.VSCODE_QUALITY }}-$(VSCODE_ARCH)-$(date +%s).tar.gz"
         mkdir -p $(dirname $ARCHIVE_PATH)
-        tar -czf $ARCHIVE_PATH -C .. VSCode-linux-$(VSCODE_ARCH)
         echo "##vso[task.setvariable variable=CLIENT_PATH]$ARCHIVE_PATH"
       env:
         GITHUB_TOKEN: "$(github-distro-mixin-password)"
       displayName: Build client
 
+    - ${{ if ne(parameters.VSCODE_CIBUILD, true) }}:
```

Comment:
Is this condition needed ? When `Build Client` is executed I assume we would want the CLI bundled in it as well ?